lib/dataHelper.tsx

```diff
@@ -0,0 +1,11 @@
+import Moment from 'moment';
```

Comment:
name files other than components in kebab case (date-helper.tsx). This eliminates any case issues with file names